generate-stackbrew-library.sh

```diff
@@ -11,7 +11,9 @@ array_10='10 dubnium'
 # shellcheck disable=SC2034
 array_12='12 erbium lts'
 # shellcheck disable=SC2034
-array_14='14 latest current'
+array_14='14'
+# shellcheck disable=SC2034
+array_15='15 latest current'
```

Comment:
I can't remember what the difference is for `latest` vs `current`. I know `current` comes from https://github.com/nodejs/Release, but I don't remember where `latest` comes from

Reply:
`latest` is a docker thing